brave/browser-laptop
# Windows style updates

Windows style updates
- Increases padding above menu by 2px

Windows 10 specific updates:
- window border was changed from black to the default blue accent color
- inactive windows now have their border set to gray

Fixes https://github.com/brave/browser-laptop/issues/4648

## Auditors
@bradleyrichter - see screenshots 
@bbondy, @bridiver - is this new window state/action ok? Since we'd like to migrate to appState, it can be done there too... but change is harder to make since focus/blur state for ALL windows needs to be tracked

## Test plan
1. Launch Brave on Windows 10 and toggle the program menu 
2. Notice that the height is 2px taller than before
3. Using the menu, open a new window
4. Notice that your new active window has a blue border and that the previous window has a gray border color

Screenshots:
**extra padding over menu**
![screen shot 2016-10-17 at 1 44 47 am](https://cloud.githubusercontent.com/assets/4733304/19431013/76c7f846-940b-11e6-8f90-32db331ac980.png)

**new blue border + gray inactive border**
![screen shot 2016-10-17 at 1 45 19 am](https://cloud.githubusercontent.com/assets/4733304/19431029/85fcaece-940b-11e6-9b1e-6652bba4b813.png)
